eventtracking/processors/whitelist.py

```diff
@@ -0,0 +1,21 @@
+"""Filter out events whose names aren't on a pre-configured whitelist"""
+
+from eventtracking.processors.exceptions import EventEmissionExit
+
+
+class WhitelistProcessor(object):
```

Comment:
Speculation:  do we expect whitelist processing to only involve the event name?  If not, I'd suggest making the name more specific now:  something like EventNameWhitelistProcessor.  I guess I was imagining that one might only allow certain event_source values as well...